lib/privatemessages/onMessage.js

```diff
@@ -34,19 +34,20 @@ exports.func = function (args) {
   const onMessage = new events.EventEmitter()
   let waitingForRequest = false
   let latest
-  getMessages({ jar, messageTab: 'Inbox', pageNumber: 0 })
+  getMessages({ jar: jar, messageTab: 'Inbox', pageNumber: 0, pageSize: 1 })
     .then(function (initial) {
       latest = initial.collection[0] ? initial.collection[0].id : 0
-      const notifications = onNotification({ jar })
+      const notifications = onNotification({ jar: jar })
```

Comment:
I note in several cases you have changed the code from `{jar}`, which works, to explicitly specifying the jar value. Is there a reason for this?

Reply:
I just did that because it makes more sense to me when getting the jar value.